Dockerfile

```diff
@@ -1,18 +1,17 @@
-FROM ruby:2.7
+FROM ruby:2.7.6
 
 RUN apt-get install -y curl \
   && curl -sL https://deb.nodesource.com/setup_18.x | bash - \
   && apt-get install -y nodejs \
   && curl -L https://www.npmjs.com/install.sh | sh
 
-RUN gem install  bundler:1.17.2 jekyll 
+RUN gem install bundler:1.17.2 jekyll:3.9.2
```

Comment:
Hmm, I don’t understand why we need to lock the version here if it is already set here:

https://github.com/scala/scala.epfl.ch/blob/d095d8fd668eed52242ddc584c7730fc5e04a1b4/Gemfile.lock#L37

Reply:
Actually good call. Why do we even need to install jekyll here. Lemme check something.